playbook/app/pb_kits/playbook/pb_collapsible/docs/_collapsible_custom_main.jsx

```diff
@@ -10,7 +10,7 @@ const CollapsibleCustomMain = () => {
         collapsed={isCollapsed}
     >
           <Background 
-              backgroundColor="white" 
+              backgroundColor="clear" 
```

Comment:
Hey @anthonymig88 , so "clear" is not one of the options available to the Background kit, so if you inspect the DOM you'll see that it's not adding any CSS for the className this is generating. I think what is happening is that since there is no CSS, it is defaulting to no color which is why this is appearing to fix the issue.
For Nitro, devs will just copy our code snippets and use them in their code so we should avoid  putting in anything that isn't actually supported by the system. 
You could try something [like this](https://github.com/powerhome/playbook/blob/9e4c30ce392d2af290471b2fb2346d27ec5f0933/playbook-website/app/javascript/components/Website/src/layouts/LayoutRight.tsx#L9) for this usecase

Let me know if you have any issues/questions here!